### PR TITLE
Move Robert Dyas to hardware

### PR DESCRIPTION
**Pull request by peternewman**
The first line of https://wiki.openstreetmap.org/wiki/Tag:shop=hardware says "sometimes known as an ironmongers shop in the UK", some of the shops are branded as "Robert Dyas The Ironmongers".

### CODE REVIEW

File: brands/shop/hardware.json [hunk 1]
```diff
@@ -83,6 +83,20 @@
         "shop": "hardware"
       }
     },
+    {
+      "displayName": "Robert Dyas",
+      "id": "robertdyas-6506bb",
+      "locationSet": {"include": ["gb"]},
+      "matchTags": ["shop/electronics"],
+      "oldid": "shop/hardware|Robert Dyas",
```
Comment (peternewman) [inline brands/shop/hardware.json:91]: Should I have changed this or left it as shop/electronics?

**Comment by peternewman** [comment]
I assume this will be enough to get iD to update the other instances as people find them?

File: brands/shop/hardware.json [hunk 2]
```diff
@@ -83,6 +83,20 @@
         "shop": "hardware"
       }
     },
+    {
+      "displayName": "Robert Dyas",
+      "id": "robertdyas-6506bb",
```
Comment (peternewman) [inline brands/shop/hardware.json:88]: Do I need to change this ID at all?

Comment (bhousel) [reply brands/shop/hardware.json:88]: No need, the build script will take care of it.

**Comment by bhousel** [comment]
> I assume this will be enough to get iD to update the other instances as people find them?

Yes 👍

File: brands/shop/hardware.json (hunk 1 above)
Comment (bhousel) [reply brands/shop/hardware.json:91]: It doesn't really matter, the `oldid` only exists as a claim on Wikidata, and I'll probably end up bulk updating them anyway.

**Comment by peternewman** [comment]
Great, thanks for that @bhousel

**Pull request merged**